cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/transformers/TapChangerBuilder.java

```diff
@@ -0,0 +1,54 @@
+package com.powsybl.cgmes.conversion.elements.transformers;
```

Comment:
Add copyright (2020)

Reply:
Done. I have not change `TapChanger` to` CgmesTapChanger` as it is used also in interpreted and converted models and will be confusing.